### DISCUSSION TITLE
error 404 when install boost by vcpkg

### DISCUSSION THREAD

**Discussion by CruzChew** [microsoft/vcpkg#32908] [Q&A]
PS D:\Tools\vcpkg\vcpkg> .\vcpkg install boost-asio:x64-windows
Computing installation plan...
The following packages will be built and installed:
  * boost-algorithm:x64-windows -> 1.82.0#2
  * boost-align:x64-windows -> 1.82.0#2
  * boost-array:x64-windows -> 1.82.0#2
    boost-asio:x64-windows -> 1.82.0#3
  * boost-assert:x64-windows -> 1.82.0#2
  * boost-bind:x64-windows -> 1.82.0#2
  * boost-build:x64-windows -> 1.82.0#2
  * boost-chrono:x64-windows -> 1.82.0#2
  * boost-concept-check:x64-windows -> 1.82.0#2
  * boost-config:x64-windows -> 1.82.0#2
  * boost-container:x64-windows -> 1.82.0#2
  * boost-container-hash:x64-windows -> 1.82.0#2
  * boost-context:x64-windows -> 1.82.0#2
  * boost-conversion:x64-windows -> 1.82.0#2
  * boost-core:x64-windows -> 1.82.0#2
  * boost-coroutine:x64-windows -> 1.82.0#2
  * boost-date-time:x64-windows -> 1.82.0#2
  * boost-describe:x64-windows -> 1.82.0#2
  * boost-detail:x64-windows -> 1.82.0#2
  * boost-exception:x64-windows -> 1.82.0#2
  * boost-function:x64-windows -> 1.82.0#2
  * boost-function-types:x64-windows -> 1.82.0#2
  * boost-functional:x64-windows -> 1.82.0#2
  * boost-fusion:x64-windows -> 1.82.0#2
  * boost-integer:x64-windows -> 1.82.0#2
  * boost-intrusive:x64-windows -> 1.82.0#2
  * boost-io:x64-windows -> 1.82.0#2
  * boost-iterator:x64-windows -> 1.82.0#2
  * boost-lexical-cast:x64-windows -> 1.82.0#2
  * boost-modular-build-helper:x64-windows -> 1.82.0#4
  * boost-move:x64-windows -> 1.82.0#2
  * boost-mp11:x64-windows -> 1.82.0#2
  * boost-mpl:x64-windows -> 1.82.0#2
  * boost-numeric-conversion:x64-windows -> 1.82.0#2
  * boost-optional:x64-windows -> 1.82.0#2
  * boost-pool:x64-windows -> 1.82.0#2
  * boost-predef:x64-windows -> 1.82.0#2
  * boost-preprocessor:x64-windows -> 1.82.0#2
  * boost-range:x64-windows -> 1.82.0#2
  * boost-ratio:x64-windows -> 1.82.0#2
  * boost-rational:x64-windows -> 1.82.0#2
  * boost-regex:x64-windows -> 1.82.0#2
  * boost-smart-ptr:x64-windows -> 1.82.0#2
  * boost-static-assert:x64-windows -> 1.82.0#2
  * boost-system:x64-windows -> 1.82.0#2
  * boost-throw-exception:x64-windows -> 1.82.0#2
  * boost-tokenizer:x64-windows -> 1.82.0#2
  * boost-tuple:x64-windows -> 1.82.0#2
  * boost-type-traits:x64-windows -> 1.82.0#2
  * boost-typeof:x64-windows -> 1.82.0#2
  * boost-unordered:x64-windows -> 1.82.0#2
  * boost-utility:x64-windows -> 1.82.0#2
  * boost-variant2:x64-windows -> 1.82.0#2
  * boost-winapi:x64-windows -> 1.82.0#2
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-windows...
-- Setting "HTTP_PROXY" environment variables to "http://127.0.0.1:26501".
Restored 0 package(s) from C:\Users\myacc\AppData\Local\vcpkg\archives in 3.98 ms. Use --debug to see more details.
Installing 1/54 boost-config:x64-windows...
Building boost-config:x64-windows...
-- Downloading https://github.com/boostorg/config/archive/boost-1.82.0.tar.gz -> boostorg-config-boost-1.82.0.tar.gz...
[DEBUG] To include the environment variables in debug output, pass --debug-env
[DEBUG] Trying to load bundleconfig from D:\Tools\vcpkg\vcpkg\vcpkg-bundle.json
[DEBUG] Failed to open: D:\Tools\vcpkg\vcpkg\vcpkg-bundle.json
[DEBUG] Bundle config: readonly=false, usegitregistry=false, embeddedsha=nullopt, deployment=Git, vsversion=nullopt
[DEBUG] Metrics enabled.
[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Feature flag 'registries' unset
[DEBUG] Feature flag 'versions' unset
[DEBUG] Feature flag 'dependencygraph' unset
Downloading https://github.com/boostorg/config/archive/boost-1.82.0.tar.gz
error: Failed to download from mirror set
error: https://github.com/boostorg/config/archive/boost-1.82.0.tar.gz: failed: status code 404
[DEBUG] D:\a\_work\1\s\src\vcpkg\base\downloads.cpp(1045):
[DEBUG] Time in subprocesses: 0us
[DEBUG] Time in parsing JSON: 5us
[DEBUG] Time in JSON reader: 0us
[DEBUG] Time in filesystem: 5479us
[DEBUG] Time in loading ports: 0us
[DEBUG] Exiting after 4.9 s (4802507us)

CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:32 (message):

      Failed to download file with error: 1
      If you are using a proxy, please check your proxy setting. Possible causes are:

      1. You are actually using an HTTP proxy, but setting HTTPS_PROXY variable
         to `https://address:port`. This is not correct, because `https://` prefix
         claims the proxy is an HTTPS proxy, while your proxy (v2ray, shadowsocksr
         , etc..) is an HTTP proxy. Try setting `http://address:port` to both
         HTTP_PROXY and HTTPS_PROXY instead.

      2. If you are using Windows, vcpkg will automatically use your Windows IE Proxy Settings
         set by your proxy software. See https://github.com/microsoft/vcpkg-tool/pull/77
         The value set by your proxy might be wrong, or have same `https://` prefix issue.

      3. Your proxy's remote server is out of service.

      If you've tried directly download the link, and believe this is not a temporary
      download server failure, please submit an issue at https://github.com/Microsoft/vcpkg/issues
      to report this upstream download server failure.

Call Stack (most recent call first):
  scripts/cmake/vcpkg_download_distfile.cmake:270 (z_vcpkg_download_distfile_show_proxy_and_fail)
  scripts/cmake/vcpkg_from_github.cmake:106 (vcpkg_download_distfile)
  ports/boost-config/portfile.cmake:3 (vcpkg_from_github)
  scripts/ports.cmake:147 (include)

error: building boost-config:x64-windows failed with: BUILD_FAILED
Elapsed time to handle boost-config:x64-windows: 5.2 s
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
    https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+boost-config
You can submit a new issue at:
    https://github.com/microsoft/vcpkg/issues/new?title=[boost-config]+Build+error&body=Copy+issue+body+from+D%3A%2FTools%2Fvcpkg%2Fvcpkg%2Finstalled%2Fvcpkg%2Fissue_body.md

**Comment by Osyotr** [comment]
> -- Setting "HTTP_PROXY" environment variables to "http://127.0.0.1:26501/".

Well it looks like a problem with your proxy because otherwise URL is accessible.
